docs/resources/waf_policy.md

```diff
@@ -0,0 +1,113 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "netlify_waf_policy Resource - netlify"
+subcategory: ""
+description: |-
+  Netlify Web Application Firewall (WAF) policy. Read more https://docs.netlify.com/security/secure-access-to-sites/web-application-firewall/
+---
+
+# netlify_waf_policy (Resource)
+
+Netlify Web Application Firewall (WAF) policy. [Read more](https://docs.netlify.com/security/secure-access-to-sites/web-application-firewall/)
+
+## Example Usage
+
+```terraform
+resource "netlify_waf_policy" "example" {
+  team_id     = data.netlify_team.team.id
+  name        = "Terraform Ruleset"
+  description = "This is a test ruleset through Terraform"
```

Comment:
nit: this is the policy not the rule set. A policy contains a set of rule sets that each in turn contain a set of rules

Reply:
You're not wrong, but this is what netlify-react-ui sets for all its policies:

```
    "name": "Baseline Ruleset",
    "description": "Netlify maintained ruleset",
```

I copied this very early in the development and didn't think about it. I'll change it to say policy.